src/styles/_variables.scss

```diff
@@ -0,0 +1,20 @@
+$main-color: #060b35;
+$crem-color: #f3f3f3;
+$grey-colort: #616070;
```

Comment:
There is a typo in the variable name `$grey-colort`. It should be `$grey-color` to maintain consistency and avoid errors when using this variable in other files【24:0†source】.